central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -72,7 +74,15 @@ func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 
 	// if not ready, add it to the status errors
 	if readyPods < desiredPods {
-		operatorErrors = append(operatorErrors, fmt.Sprintf("compliance operator not ready.  Only %d pods are ready when %d are desired.", readyPods, desiredPods))
+		operatorErrors = append(operatorErrors, fmt.Sprintf("Compliance operator not ready. Only %d out of desired %d are ready.", readyPods, desiredPods))
```

Comment:
Provided this gets displayed in the UI, let's make it a sentence by adding a verb in the first part, and mention what object is meant in the second part.
```suggestion
		operatorErrors = append(operatorErrors, fmt.Sprintf("Compliance operator is not ready. Only %d pods out of desired %d are ready.", readyPods, desiredPods))
```